package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "blueprint-markdown-renderer",
   "nicename": "Blueprint Markdown Renderer",
-  "version": "1.2.0",
+  "version": "0.0.0-development",
```

Comment:
But using fix/feat/breaking in commit messages should help and solve it for us.
Hopefully we won't run into huge numbers too soon :-)